upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.16-v1.12.yaml.template

```diff
@@ -817,6 +819,26 @@ spec:
 {{ end }}
       hostNetwork: true
       initContainers:
+      {{- if semverCompare ">=1.12.8" $semver }}
```

Comment:
`{{- if semverCompare "~1.11.15 || ~1.12.8 || >= 1.13.1" $semver }}`

Reply:
Thanks for the suggestion, I have made the modification. :slightly_smiling_face: